src/components/organisms/SectionContents/experts.js

```diff
@@ -1,19 +1,25 @@
 //** Standard Frameworks */
-import * as React from 'react'
+import * as React from 'react';
 
 //** Components */
 //** Atoms */
 import ModalBtn from "../../atoms/ModalButton";
 import Alert from "../../atoms/Alert";
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Experts
  */
```

Comment:
This is disgusting drug-induced crap, and should die 
To put it with Linus words.